modules/administration-guide/pages/configuring-microsoft-visual-studio-code.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: CONCEPT
+:description: Configuring Microsoft Visual Studio Code
```

Comment:
on https://github.com/microsoft/vscode it is called -  Visual Studio Code - Open Source ("Code - OSS")
let's use this wording and drop Microsoft

Reply:
VS Code has so many variation of its name that I'm considering contacting the legal team to get a definitive answer, to be honest.

Reply:
please, refer to the https://github.com/microsoft/vscode README
this is an upstream version of `che-code` which must be referred as `Visual Studio Code - Open Source ("Code - OSS")` as stated in the documentation

Reply:
done